proto/viam/app/data/v1/data.proto

```diff
@@ -179,6 +189,22 @@ message BinaryDataByIDsResponse {
   uint64 count = 2;
 }
 
+// BoundingBox represents a labeled bounding box on an image.
+// x and y values are normalized ratios between 0 and 1.
+message BoundingBox {
+  string id = 1;
+  string label = 2;
+  float normalized_x_min = 3;
```

Comment:
[supernit]: I think `x_min_normalized` is a bit more readable since it's then formatted as thing_attribute